swift_scality_backend/utils.py

```diff
@@ -118,3 +126,35 @@ def is_sproxyd_conf_valid(conf):
 
     raise SproxydConfException("Make sure by_path_enabled is set "
                                "and check Sproxyd logs.")
+
+
+@contextlib.contextmanager
+def import_specific(*requirements):
```

Comment:
I'd add some logging here (or in `get_urllib3`) if some special measures need to be taken to load a 'correct' urllib3.